### DISCUSSION TITLE
Question re filter matrices

### DISCUSSION THREAD

**Discussion by UkooLabs** [SixLabors/ImageSharp#1367] [Q&A] [answered]
Was wondering the following If you have a filter matrix that say does sepia and another that does some other filter, can you multiply the matrices to get a combined effect and if so would be a cool processor if you could combine filter matrices and apply.

**Comment by JimBobSquarePants** [comment, marked as answer]
Yeah you can already [multiply](https://docs.sixlabors.com/api/ImageSharp/SixLabors.ImageSharp.ColorMatrix.html#SixLabors_ImageSharp_ColorMatrix_op_Multiply_SixLabors_ImageSharp_ColorMatrix_SixLabors_ImageSharp_ColorMatrix_) `ColorMatrix` structs and pass them to `Filter`.  

Here's an example from our unit tests.

```c#
ColorMatrix brightness = KnownFilterMatrices.CreateBrightnessFilter(0.9F);
ColorMatrix hue = KnownFilterMatrices.CreateHueFilter(180F);
ColorMatrix saturation = KnownFilterMatrices.CreateSaturateFilter(1.5F);
return brightness * hue * saturation;
```
